test/WebJobs.Script.Tests/MetricsEventManagerTests.cs

```diff
@@ -72,20 +72,6 @@ public class MetricsEventManagerTests
                 string.Format("There are events with different execution id. List:{0} Invalid entries:{1}",
                     SerializeFunctionExecutionEventArguments(argsList),
                     SerializeFunctionExecutionEventArguments(invalidArgsList)));
-
```

Comment:
This change would require tweaks in the validation logic as the number and types of ETW events have changed. I didn't want to spend much time tweaking this validation routine, I would introduce this back in a separate commit after making it work with this change.